src/admin/pages/VendorAccountFields.vue

```diff
@@ -49,7 +49,24 @@
                         :class="{'dokan-form-input': true, 'has-error': getError('store_name')}"
                         :placeholder="getError( 'store_name' ) ? __( 'Store Name is required', 'dokan-lite' ) : __( 'Store Name', 'dokan-lite' )">
                 </div>
-
+                <div class="column">
```

Comment:
Can you please try to create a component for store category so that we can use this from both places?